### PR TITLE
Bug 1085718 - FTU: selected language should be visible (r=sfoster)

### PR DESCRIPTION
**Pull request by autonome**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[autonome Dietrich Ayala (autonome)](https://github.com/autonome) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=45973c87719d)

File: apps/ftu/js/language.js [hunk 1]
```diff
@@ -44,13 +44,17 @@ var LanguageManager = {
...
+    var selectedLanguageNode = null;
     LanguageList.get(function fillLanguageList(allLanguages, currentLanguage) {
       for (var lang in allLanguages) {
         var input = document.createElement('input');
         input.type = 'radio';
         input.name = 'language.current';
         input.value = lang;
-        input.checked = (lang === currentLanguage);
+        if (lang === currentLanguage) {
+          input.checked = (lang === currentLanguage);
```
Comment (sfoster) [inline apps/ftu/js/language.js]: Nit: Seems redundant, we know lang == currentLanguage, so maybe just use true here

Comment (autonome) [inline apps/ftu/js/language.js]: oops, refactoring miss!

**Comment by try-server-hook** [comment]
[autonome Dietrich Ayala (autonome)](https://github.com/autonome) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b28c162942ef)

**Pull request closed without merging**